app/javascript/packages/components/text-input.tsx

```diff
@@ -32,6 +37,9 @@ function TextInput(
       <label className="usa-label" htmlFor={inputId}>
         {label}
       </label>
+      <div id="kvgx-hint" className="usa-hint">
```

Comment:
in validated field component we append `unique-id` to the hint (the same id appended to the error). Maybe we want to do something similar here?

Reply:
Yeah, I agree with @svalexander . React 18 would have [`useId`](https://reactjs.org/docs/hooks-reference.html#useid) for this, but we're not able to upgrade yet due to browser support constraints. In the meantime, we have our own version of this that can be used, [`useInstanceId`](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/react-hooks/use-instance-id.ts).

```ts
import { useInstanceId } from '@18f/identity-react-hooks';
```

Reply:
IDs must be unique, and since a React component could be rendered multiple times on the same page, you'd want to make sure that each has their own ID.

Reply:
Oh, you’re missing ‘aria-describedby’ on the input, which would create the relationship between the input and hint using the hint’s ID.